pacemaker.spec.in

```diff
@@ -169,7 +170,7 @@ It will run scripts at initialization, when machines go up or down,
 when related resources fail and can be configured to periodically check
 resource health.
 
-Available rpmbuild rebuild options:
+Available rpmbuild rebuild options (not necessarily applicable/with effect):
```

Comment:
Alternatively, specfile should pass `--with-cman` if the same was passed to `rpmbuild` and `configure` script should fail if this requirement cannot be fulfilled. The same then ensured for other rpmbuild options and then I would agree the note makes no sense, at least for those that can figure out what's the default selection from `bcond` statements.